ioctl/util/crypt.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package util
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/base64"
+	"fmt"
+	"io"
+)
+
+// hashTo32Bytes will compute a cryptographically useful hash of the input string.
+func hashTo32Bytes(input string) []byte {
+
+	data := sha256.Sum256([]byte(input))
+	return data[0:]
+
+}
+
+// DecryptString Takes two strings, cryptoText and keyString.
+// cryptoText is the text to be decrypted and the keyString is the key to use for the decryption.
+// The function will output the resulting plain text string with an error variable.
+func DecryptString(cryptoText string, keyString string) (plainTextString string, err error) {
+
+	encrypted, err := base64.URLEncoding.DecodeString(cryptoText)
+	if err != nil {
+		return "", err
+	}
+	if len(encrypted) < aes.BlockSize {
+		return "", fmt.Errorf("cipherText too short. It decodes to %v bytes but the minimum length is 16", len(encrypted))
+	}
+
+	decrypted, err := decryptAES(hashTo32Bytes(keyString), encrypted)
+	if err != nil {
+		return "", err
+	}
+
+	return string(decrypted), nil
+}
+
+func decryptAES(key, data []byte) ([]byte, error) {
+	// split the input up in to the IV seed and then the actual encrypted data.
+	iv := data[:aes.BlockSize]
+	data = data[aes.BlockSize:]
+
+	block, err := aes.NewCipher(key)
+	if err != nil {
+		return nil, err
+	}
+	stream := cipher.NewCFBDecrypter(block, iv)
+
+	stream.XORKeyStream(data, data)
+	return data, nil
+}
+
+// EncryptString Takes two string, plainText and keyString.
+// plainText is the text that needs to be encrypted by keyString.
+// The function will output the resulting crypto text and an error variable.
+func EncryptString(plainText string, keyString string) (cipherTextString string, err error) {
+
+	key := hashTo32Bytes(keyString)
+	encrypted, err := encryptAES(key, []byte(plainText))
+	if err != nil {
+		return "", err
+	}
+
+	return base64.URLEncoding.EncodeToString(encrypted), nil
```

Comment:
i think don't need an extra base64 encoding?

the result of EncryptString is written to the config file, while ioutil.WriteFile() take []byte as input, so can directly use return result from encryptAES